MetaMask/logo
# Restructure project to match others

Move implementation files to `src/`, move `convert.js` to `scripts/`, and move data files to `data/`.

This matches how our other libraries tend to be structured.

---

To test this:

- Run `yarn lint`. This should pass.
- Remove `fox.json` from `data`, then run `yarn convert`. The file should be regenerated (and unchanged).
- Run `yarn demo --all`, then open `demo/dist/index.html`. Click on each demo; you should see the 3d fox, and it should move when you move your mouse.
- Run `yarn pack`. Uncompress `metamask-logo-v<version>.tgz`, and open the resulting `package` directory. You should see `src/index.js`, `src/util.js`, and `data/fox.json`, among the README, LICENSE, CHANGELOG, and `package.json`.
